source/deploying-with-puppet/setup-puppet/install-puppet-agent.rst

```diff
@@ -8,26 +8,50 @@ Installing Puppet agent
 In this section we assume you have already installed the ``apt`` or ``yum`` Puppet repository on your agent system in the same way that you did on your Puppet Server.
 
 Installation on CentOS/RHEL/Fedora
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+------------------------------------
+
+Install the Puppet yum repository and then the "puppet-agent" package. See this `index <https://yum.puppetlabs.com/>`_ to find the correct rpm file needed to install the puppet repo for your Linux distribution. For example, to install Puppet 5 for CentOS 7 or RHEL 7, do the following:
 
 .. code-block:: console
 
-   # yum install puppet
-   # puppet resource package puppet ensure=latest
+   # rpm -ivh https://yum.puppetlabs.com/puppet5/puppet5-release-el-7.noarch.rpm
+   # yum -y install puppet-agent
+
+.. note:: 
+
+  For a correct installation we recommend the use of Puppet versions equal or greater than 5. 
 
 Installation on Debian/Ubuntu
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+------------------------------
+
+Install ``curl``, ``apt-transport-https`` and ``lsb-release``:
 
 .. code-block:: console
 
-   # apt-get install puppet
-   # apt-get update
-   # puppet resource package puppet ensure=latest
+	# apt-get update
+	# apt-get install curl apt-transport-https lsb-release
+
+Get the appropriate Puppet apt repository, and then the "puppet-agent" package. See https://apt.puppetlabs.com to find the correct deb file to install the puppet repo for your Linux distribution.
+
+.. code-block:: console
+
+  # wget https://apt.puppetlabs.com/puppet5-release-xenial.deb
+  # dpkg -i puppet5-release-xenial.deb
+  # apt update
+  # apt-get install -y puppet-agent
+
+.. note:: 
+  
+  For a correct installation we recommend the use of Puppet versions equal or greater than 5. The releases supported by the manifest to install Wazuh are as follows: 
```

Comment:
This looks like 2 notes:
- For a correct installation we recommend the use of Puppet versions equal or greater than 5.
- The releases supported by the manifest to install Wazuh are as follows: